install/streams.csv

```diff
@@ -2013,10 +2013,10 @@ TLED,20,200,true,false,true,2010-09-05T05:00:02Z,2012-02-21T01:00:00Z
 TLZ,10,1,false,false,false,2009-05-07T01:00:00Z,9999-01-01T00:00:00Z
 TLZ,10,100,false,false,false,2009-05-07T01:00:00Z,9999-01-01T00:00:00Z
 TLZ,20,1,false,false,false,2009-05-07T01:00:00Z,2016-02-12T01:30:00Z
-TLZ,20,50,false,false,false,2009-05-07T01:00:00Z,2020-01-15T02:12:00Z
-TLZ,20,200,false,false,true,2009-05-07T01:00:00Z,2020-01-15T02:12:00Z
-TLZ,21,50,false,false,false,2020-01-15T02:16:00Z,2020-06-16T00:00:00Z
-TLZ,21,200,false,false,true,2020-01-15T02:16:00Z,2020-07-02T02:31:00Z
+TLZ,20,50,false,false,false,2009-05-07T01:00:00Z,2016-02-12T01:30:00Z
+TLZ,20,200,false,false,true,2009-05-07T01:00:00Z,2016-02-12T01:30:00Z
+TLZ,21,50,false,false,false,2016-02-12T01:30:01Z,2020-06-16T00:00:00Z
+TLZ,21,200,false,false,true,2016-02-12T01:30:01Z,2020-07-02T02:31:00Z
```

Comment:
It looks like 200hz triggered ended a bit earlier.
200hz continuous seems to start on 2020.040 (Feb9).
```
murieln@ubuntu:~/Desktop/Shared/sh$ ./list_miniseed.sh TLZ 2020 040 050 | grep HNN | sed -e 's/"//g' -e 's/Key//' -e 's/://'
  miniseed/2020/2020.040/TLZ.NZ/2020.040.TLZ.21-HNN.NZ.D
  miniseed/2020/2020.040/TLZ.NZ/2020.040.TLZ.21-HNN.NZ.D
  miniseed/2020/2020.041/TLZ.NZ/2020.041.TLZ.21-HNN.NZ.D
  miniseed/2020/2020.042/TLZ.NZ/2020.042.TLZ.21-HNN.NZ.D
  miniseed/2020/2020.043/TLZ.NZ/2020.043.TLZ.21-HNN.NZ.D
  miniseed/2020/2020.044/TLZ.NZ/2020.044.TLZ.21-HNN.NZ.D
  miniseed/2020/2020.045/TLZ.NZ/2020.045.TLZ.21-HNN.NZ.D
  miniseed/2020/2020.046/TLZ.NZ/2020.046.TLZ.21-HNN.NZ.D
  miniseed/2020/2020.047/TLZ.NZ/2020.047.TLZ.21-HNN.NZ.D
  miniseed/2020/2020.048/TLZ.NZ/2020.048.TLZ.21-HNN.NZ.D
  miniseed/2020/2020.049/TLZ.NZ/2020.049.TLZ.21-HNN.NZ.D
  miniseed/2020/2020.050/TLZ.NZ/2020.050.TLZ.21-HNN.NZ.D
```